src/pwm.rs

```diff
@@ -304,6 +311,10 @@ macro_rules! small_timer {
                     tim.ccmr1_output().modify(|_, w| unsafe { w.oc1pe().set_bit().oc1m().bits(6) });
                 }
 
+                // if PINS::C2 {
```

Comment:
Remove this if its note needed.

Reply:
The commented lines are due to incompatibility issues with the PAC that does not have all the registers of the MCU yet, so once the PAC is updated with support these can be un-commented and all will start to work.